python/electrostatic_force.py

```diff
@@ -0,0 +1,362 @@
+from tkinter import *
+import compute
+def com1():
+    global lavel
+    charge1val=Charge1.get()
+    #print(charge1val)
+    charge2val=Charge2.get()
+    #print(charge2val)
+    root.destroy()
+    root1=Tk()
+    global flag
```

Comment:
Codacy found an issue: [Global variable 'flag' undefined at the module level](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279653)